index.html

```diff
@@ -84,7 +83,163 @@ <h1 class="primary py-2 my-2">
       </div>
     </section>
 
-    <
+    <section class="grid text-center m-4 p-2 projects">
+      <h1 class="primary py-2 my-4 text-center">My Recent Work</h1>
```

Comment:
- [ ] KIndly make sure to follow the styles on provided Figma design. Please use the same typography styles while styling the `My Recent Work` title. :wink: 

   - ### Your design
   ![image](https://user-images.githubusercontent.com/87197412/200126946-86cc384f-9fd4-440e-829d-a305e3912e9e.png)

  - ### Figma design
    ![image](https://user-images.githubusercontent.com/87197412/200126960-d87b9882-e7a8-47fc-a0a2-86378c9c4403.png)

Reply:
Fixed
![image](https://user-images.githubusercontent.com/44822765/200138390-d07b14b3-e0c4-48b6-9dc4-4cc7bed1841d.png)